collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_fs.py

```diff
@@ -72,6 +75,13 @@
     required: false
     type: bool
     default: false
+  smb_aclmode:
+    description:
+      - Define whether to SMB protocol is enabled for the filesystem.
```

Comment:
Suggest to change to:
`Specify the ACL mode for the SMB protocol as either shared with NFS or native`